WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -174,22 +188,6 @@ - (void)viewWillAppear:(BOOL)animated
     [self reloadData];
 }
 
-- (void)viewWillDisappear:(BOOL)animated
-{
-    [super viewWillDisappear:animated];
-
-    if (self.isStandalone) {
-        if ((self.isBeingDismissed || self.parentViewController.isBeingDismissed) && !self.isStandaloneEditorDismissingAfterSave) {
-            // TODO: Implement it using a ViewModel or a child context to eliminate the risk of accidently saving the changes without uploading them
-            [self.apost.managedObjectContext refreshObject:self.apost mergeChanges:NO];
```

Comment:
To discard the changes, you no longer need to do anything thanks to the child context.
To save, you now need to tap "Done" or "Save".